src/Microsoft.Extensions.Configuration.AzureAppConfiguration/TracingUtils.cs

```diff
@@ -23,14 +22,13 @@ internal static class TracingUtils
         public static string GenerateUserAgent(string currentUserAgent = null)
         {
             Assembly assembly = typeof(AzureAppConfigurationOptions).Assembly;
-            var userAgent = new StringBuilder($"{assembly.GetName().Name}/{assembly.GetName().Version}");
+            string informationalVersion = assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;
```

Comment:
Just to understand, who will set the `AssemblyInformationalVersionAttribute`?

Reply:
This attribute is automatically generated by the `.NET Core SDK` based on the `Version` property in the `csproj` file. When I build the project, I can see the `*AssemblyInfo.cs` file in the obj directory with the `AssemblyInformationalVersionAttribute` attribute along with other auto-generated attributes, and these are also applied to the corresponding assembly. More information can be found [here](https://compiledexperience.com/blog/posts/assemblyinfo-generation).

Reply:
Thanks.